src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './App.css';
+import ToDo from './containers/ToDo';
```

Comment:
Always separate components imports  and CSS imports with new line and format `imports` in a ladder form.